### PR TITLE
add LowPowerTicker examples

### PR DESCRIPTION
**Pull request by maciejbocianski**
Move examples:
https://os.mbed.com/teams/mbed_example/code/LowPowerTimeout-example/
https://os.mbed.com/teams/mbed_example/code/LowPowerTimer-example/
https://os.mbed.com/teams/mbed_example/code/LowPowerTicker_HelloWorld/
to this repo

Update example code to latest API's

Docs update PR: https://github.com/ARMmbed/mbed-os-5-docs/pull/1252

### CODE REVIEW

File: APIs_Drivers/LowPowerTimeout-example/main.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (jamesbeyond) [inline APIs_Drivers/LowPowerTimeout-example/main.cpp]: could you rename the folder to the example?
something like `lowpowerTimeout_ex1` ?

File: APIs_Drivers/LowPowerTimer-example/main.cpp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*
```
Comment (jamesbeyond) [inline APIs_Drivers/LowPowerTimer-example/main.cpp]: could you rename the folder to the example?
something like `lowpowerTimer_ex1` ?

File: APIs_Drivers/LowPowerTicker_HelloWorld/main.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/*
```
Comment (jamesbeyond) [inline APIs_Drivers/LowPowerTicker_HelloWorld/main.cpp]: could you rename the folder to the example?
something like `lowpowerTicker_ex1`?

**Comment by maciejbocianski** [comment]
@jamesbeyond examples names updated

**Comment by maciejbocianski** [comment]
ping @jamesbeyond

**Pull request merged**